doc/missing-vz-api.md

```diff
@@ -0,0 +1,58 @@
+# Missing API
+
+This document contains a non-exhaustive list of API which are not used/supported by `vfkit`, but which would be useful to have.
+
+## non-vz API
+
+- start vfkit process (integrating with https://pkg.go.dev/os/exec )
+- get VM IP address [https://github.com/code-ready/crc/blob/0d76300c1a618598c209bab32a8deb4ca6c2d8c6/pkg/drivers/vfkit/network_darwin.go#L54-L59]
+
+## [vz](https://pkg.go.dev/github.com/Code-Hex/vz/v2) API
```

Comment:
Do we really need this list because everytime we expose something on vfkit side this list need to be updated, we might forget about updating it and quickly it will become outdated.

Reply:
The document explicitly says that the list is non-exhaustive, and it's already actually missing the set of API needed for macos guests. I will prune it some more, the audio/keyboard/pointing device configuration is not that relevant for now. But I'd like to keep this doc in some form as some TODO/to be implemented list, much easier than going back to https://pkg.go.dev/github.com/Code-Hex/vz/v2 and manually removing what is already present in vfkit.

Reply:
The list is shorter in the latest push, but still present.

Reply:
I think this is kind of diff on what vfkit have and vz api have, as per @cfergeau it is good place to someone try to look into contributing and pick something from this list.